src/l10n/locale/pt.ts

```diff
@@ -1,3 +1,132 @@
 // Português
 
-export default {};
\ No newline at end of file
+export default {
+  //main.ts
+  "Open Dictionary View": "Abrir Vista do Dicionário",
+  "Open Language Switcher": "Abrir Selector de Língua",
```

Comment:
In European Portuguese is rare the use of "Selector" in soft language. Also we don't say "Língua" in this way, but "idioma".
I suggest "Selecionar Idioma".
<img width="388" alt="image" src="https://user-images.githubusercontent.com/22862501/169618053-2c10364c-4378-4b16-9e64-7b8710ab2edc.png">